.travis.yml

```diff
@@ -65,9 +65,11 @@ matrix:
 # Validate rule files
         - make validateRules
 # check htmlreport stuff
-        - ./htmlreport/test_htmlreport.py
+        - python2 htmlreport/test_htmlreport.py
```

Comment:
It might be better to run the check script after each time the test_htmlreport.py is executed. In theory if the output is invalid after the first execution we won't detect it.

Reply:
No, that's not the case, `test_htmlreport.py` and `check.sh` are independent of each other. But I figured it would make sense to use a temporary dir when running `check.sh` to avoid issues with lingering files so I added that to the PR.